client/packages/common/src/ui/components/inputs/Filters/FilterMenu.tsx

```diff
@@ -47,9 +61,18 @@ export const FilterMenu: FC<FilterDefinitions> = ({ filters }) => {
       setActiveFilters([]);
       return;
     }
-    const selectedFilter = filters.find(fil => fil.urlParameter === selected);
-    if (selectedFilter)
-      setActiveFilters(current => [...current, selectedFilter]);
+    if (selected.type === 'group') {
+      const newFilters = selected.elements.filter(
+        f =>
+          activeFilters.findIndex(
+            fil => fil.urlParameter === f.urlParameter
+          ) === -1
+      );
+      setActiveFilters(current => [...current, ...newFilters]);
+      return;
```

Comment:
If it's a filter group, pull the elements out and add them individually to the activeFilters list